static/js/floating_recipient_bar.js

```diff
@@ -85,7 +85,7 @@ export function first_visible_message(bar) {
     }
 
     // If none of the messages are visible, just take the last message.
-    return $(messages[messages.length - 1]);
+    return messages.last();
```

Comment:
What's the story with removing the `$()~ wrapper?  I can imagine it being intentional that this returns a jQuery object.

Reply:
Confirmed these are already jQuery objects.

Reply:
Right. `messages` is a jQuery object; indexing a jQuery object gives an `HtmlElement` but [`.last()`](https://api.jquery.com/last/) gives a jQuery object.